libs/api-catalogue/services/src/lib/restmetadata.service.ts

```diff
@@ -50,8 +53,12 @@ export class RestMetadataService {
         data: [],
         access: [AccessCategory.XROAD],
         type: [TypeCategory.REST],
-        environments: [Environment.DEV],
-        versions: [],
+        environments: [
+          {
+            environment: Environment.DEVELOPMENT, // TODO: Needs to be environment aware
```

Comment:
I guess this will become environment aware at a later time?

Reply:
Yes, Guðjón is working on that feature in the XRoad Collector.
So I was just making necessary changes so it wouldn't break.